src/js/components/FormField/FormField.js

```diff
@@ -214,6 +219,8 @@ const FormField = forwardRef(
     const { border: themeBorder } = formFieldTheme;
     const debounce = useDebounce();
 
+    const portalContext = useContext(PortalContext) || [];
```

Comment:
I'm wondering if we should have the default PortalContext value be `[]`, so we don't need the `|| []` here.

Reply:
done